uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/document-list.controller.js

```diff
@@ -25,6 +25,7 @@ define([
     var vm = this;
     var defaultDocumentStatus = ['1', '2']; // 1: 'awaiting upload' | 2: 'awaiting approval
 
+    vm.calledFormDocuments = true;
```

Comment:
A better name like `isDocumentSection` would be more readable. Also please add comment in the code itself, explanation what it does.

Reply:
Agreed! will fix.